packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -58,15 +57,62 @@ const SideNavCollapsedContainer = styled("div", {
     odysseyDesignTokens: DesignTokens;
     isSideNavCollapsed: boolean;
   }) => ({
-    backgroundColor: odysseyDesignTokens.HueNeutral300,
-    paddingTop: odysseyDesignTokens.Spacing5,
+    display: "flex",
+    "&:before": {
+      height: "100%",
+      width: 0,
+      content: '" "',
+    },
+    "&:has(svg:hover)": {
+      "&:before": {
+        width: isSideNavCollapsed ? "8px" : 0,
+        transitionProperty: "width, background-color",
+        transitionDuration: odysseyDesignTokens.TransitionDurationMain,
+        transitionTimingFunction: odysseyDesignTokens.TransitionTimingMain,
+        backgroundColor: isSideNavCollapsed
+          ? odysseyDesignTokens.HueNeutral200
+          : "transparent",
+      },
+    },
```

Comment:
I think, in the SVG styles, there's a way to alter them by checking if the parent is hovered with either MUI or CSS. I forget.

I won't block the PR on this, but it's something to keep in mind that there's potentially a better way of hovering the child on the parent.